usr/share/rear/finalize/default/060_compare_files.sh

```diff
@@ -0,0 +1,6 @@
+if [ -e $VAR_DIR/layout/config/files.md5sum ] ; then
+    if ! chroot $TARGET_FS_ROOT md5sum -c --quiet --warn < $VAR_DIR/layout/config/files.md5sum ; then
+        LogPrintError "Some configuration files in the restored system do not match the saved layout!"
```

Comment:
it is true that improperly formatted md5sum file should not happen, because we generated it ourselves, but as a famous person said: "trust, but verify", so I like to be prepared for the unexpected. Thanks for catching the problem with the combination of `--warn` and `--quiet`, I was not aware of it, and I will remove `--warn` for simplicity (if there is such a problem, the added check does not warrant the complication).